docs/sources/developer/architecture/components/internal-logger.md

```diff
@@ -0,0 +1,18 @@
+# Internal Logger
+
+The internal logger is a helper utility that is used to log messages from the library itself. Due to the fact that it
+relies on the unpatched console, the messages printed by the internal logger are not captured during autoinstrumentation
```

Comment:
nit: unpatched 'browser' console - (if that is what is meant by console?)
nit: autoinstrumentation -> auto-instrumentation (I might be wrong here, it may be 2 words (auto instrumenation). 
I just searched OTel docs, and they *exclusively* refer to it as "automatic instrumentation" -> https://opentelemetry.io/docs/instrumentation/java/automatic/ ).

Reply:
By `console`, it can mean Node.JS console as well given that the core package is platform agnostic.

Agree on auto-instrumentation.